build.gradle

```diff
@@ -198,6 +198,12 @@ dependencyManagement {
     dependencySet(group: 'org.mockito', version: versions.mockitoJupiter) {
       entry 'mockito-core'
     }
+    // CVE-2019-0232 - command line injections on windows
+    dependencySet(group: 'org.apache.tomcat.embed', version: '9.0.19') {
```

Comment:
maybe consider bumping the tomcat version used on the ASE too?
https://github.com/hmcts/cnp-module-webapp/blob/master/variables.tf#L154
it defaults to latest 8.x